# Description: don't use server side gradients most drivers don't handle it
--- a/src/cairo-xlib-display.c
+++ b/src/cairo-xlib-display.c
@@ -266,11 +266,7 @@
     /* Prior to Render 0.10, there is no protocol support for gradients and
      * we call function stubs instead, which would silently consume the drawing.
      */
-#if RENDER_MAJOR == 0 && RENDER_MINOR < 10
     display->buggy_gradients = TRUE;
-#else
-    display->buggy_gradients = FALSE;
-#endif
     display->buggy_pad_reflect = FALSE;
     display->buggy_repeat = FALSE;
 
